app/views/mailer/gem_pushed.html.erb

```diff
@@ -0,0 +1,20 @@
+<p>
+  A gem you are responsible for has recently been pushed to Rubygems.org.
```

Comment:
Make sure to use RubyGems.org (note casing) in the email content.

Reply:
Thanks & fixed!